plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptionsQuantitySelector.php

```diff
@@ -169,19 +147,17 @@ protected function render( $attributes, $content, $block ) {
 		$product_html = ob_get_clean();
 
 		$product_name = $product->get_name();
-		$product_html = $is_stepper_style ? $this->add_steppers( $product_html, $product_name ) : $product_html;
+		$product_html = $this->add_steppers( $product_html, $product_name );
+		$product_html = $this->add_stepper_classes( $product_html );
 
-		$parsed_attributes  = $this->parse_attributes( $attributes );
-		$product_html       = $is_stepper_style ? $this->add_stepper_classes( $product_html ) : $product_html;
 		$classes_and_styles = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes, array(), array( 'extra_classes' ) );
 
 		$classes = implode(
 			' ',
 			array_filter(
 				array(
-					'wp-block-add-to-cart-with-options-quantity-selector wc-block-add-to-cart-with-options__quantity-selector',
+					'wp-block-add-to-cart-with-options-quantity-selector wc-block-add-to-cart-with-options__quantity-selector wc-block-add-to-cart-with-options__quantity-selector--stepper',
```

Comment:
I think we can remove the `wc-block-add-to-cart-with-options__quantity-selector--stepper` class now. It's not used anywhere. :slightly_smiling_face: